vault-to-tink/README.md

```diff
@@ -6,6 +6,7 @@
 1. The kubectl context needs to be in the correct namespace (`kubectl config set-context --current --namespace <namespace>`)
 1. Docker is running and accessible without `sudo` to run the `clojure` container
 1. `jq` is installed.
+1. The API token from your Kubernetes secret is accessible to be used for the import.
```

Comment:
are we taking abt `CIRCLECI_TOKEN` here?

Reply:
It's `api-token` or `CIRCLE_INTERNAL_API_TOKEN`  or `CIRCLE_BOT_TOKEN` - it has a lot of names in our chart 😅

Reply:
and that is why, let's add this as well. `token listed here - CIRCLE_HOSTNAME/account/api`

Reply:
so let's add this as well - `kubectl get secret api-token -o yaml | yq '.data.api-token' | base64 -d`

Reply:
or may be add in the script itself rather than asking user.

Reply:
I do want to add it to the script but I figured it wasn't worth the rush to get it in before we release 4.2
I'll file a ticket for it

Reply:
https://circleci.atlassian.net/browse/ONPREM-29